cloud/new_ec2_client.go

```diff
@@ -45,6 +46,8 @@ type AWSClient interface {
 
 	// DescribeSpotPriceHistory is a wrapper for ec2.DescribeSpotPriceHistory.
 	DescribeSpotPriceHistory(*ec2.DescribeSpotPriceHistoryInput) (*ec2.DescribeSpotPriceHistoryOutput, error)
+
+	getInstanceInfo(string) (*ec2.Instance, error)
```

Comment:
mixing private and public methods in a single public interface is weird.

Reply:
done